bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -28,215 +28,150 @@ spec:
         openshift.io/component: network
         kubernetes.io/os: "linux"
     spec:
-      # Requires fairly broad permissions - ability to read all services and network functions as well
-      # as all pods.
       serviceAccountName: ovn-kubernetes-controller
       hostNetwork: true
-      hostPID: true
       priorityClassName: "system-cluster-critical"
+      # volumes in all containers:
+      # (container) -> (host)
+      # /etc/openvswitch -> /var/lib/ovn/etc - ovsdb data
+      # /var/lib/openvswitch -> /var/lib/ovn/data - ovsdb pki state
+      # /run/openvswitch -> tmpfs - sockets & pids
```

Comment:
You've confirmed it's just sockets and pids? `/run` would normally be persistent across an OVN restart, so this is a semantic change...

Reply:
Yup, but thanks for checking.